api/v1beta1/common_types.go

```diff
@@ -91,6 +91,13 @@ const (
 	// on the reconciled object.
 	PausedAnnotation = "cluster.x-k8s.io/paused"
 
+	// IsPausedAnnotation is an annotation whose value indicates whether or not
+	// a Cluster resource is fully paused. It may differ from `spec.paused` when
+	// the Cluster is in the process of pausing or unpausing. This annotation is
+	// only expected to be set by the Cluster API controller, not the user. When
+	// not set, `spec.paused` is assumed to indicate the same status.
+	IsPausedAnnotation = "cluster.x-k8s.io/is-paused"
```

Comment:
`cluster.x-k8s.io/is-paused` is a little too close to `cluster.x-k8s.io/paused` and could lead to confusion/human error.  Some alternatives (feel free to suggest more):
- `cluster.x-k8s.io/reconcile-paused`
- `cluster.x-k8s.io/controller-paused`

Alternatively, since this is very much targeting the move use case (? are there other use cases where we see this being used?), this could be something more generic than `paused`:

- `cluster.x-k8s.io/ready-for-move` 
- `cluster.x-k8s.io/paused-to-move-complete`

Reply:
Given that `clusterctl.cluster.x-k8s.io/delete-for-move` was added in https://github.com/kubernetes-sigs/cluster-api/pull/8322 what about someththing like `clusterctl.cluster.x-k8s.io/paused-for-move`?

Reply:
I'd vote for something that doesn't include "move" since while that's the main thing we're addressing, this problem exists more generally for anything else that might exist that should wait for clusters to be paused.